retrofit_gen/lib/parser/parser.dart

```diff
@@ -133,5 +133,33 @@ WriteInfo parse(ClassElement element, ConstantReader annotation) {
   }
 
   return WriteInfo(
-      element.displayName, basePath, basePathParams, baseMetaData, reqs);
+      element.displayName, basePath, basePathParams, baseMetadata, reqs);
+}
+
+String toStringValue(DartObject value) {
+  if (value.isNull) {
+    return 'null';
```

Comment:
better to return a real null here

Reply:
Why?

Reply:
Because if I have put null into my metadata I expect to get a null to do `if(meta == null) ...` by returning string "null" then `meta == null` is false

Reply:
it wont return `"null"`. it will return `null`.

Reply:
`toStringValue` is like JSON encoder. it just encodes built-in dart values in a map into dart code. It preserves type.

Reply:
I've tested you're right... still new in Dart I didn't know that looks strange to me but ok lol